### PR TITLE
added first init of psalm

### LINKED ISSUE #47: Add phpstan or psalm static code analysis

### PR DESCRIPTION
**Pull request by AlessandroMinoccheri**
This is the first runt of psalm, we can improve it and fix some issues into a next PR.

closes #47

### CODE REVIEW

**Comment by JellyBellyDev** [comment]
Thanks @AlessandroMinoccheri for taking charge of the issue #47

File: psalm.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<psalm
+    errorLevel="6"
+    resolveFromConfigFile="true"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns="https://getpsalm.org/schema/config"
+    xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
+>
+    <projectFiles>
+        <directory name="src" />
```
Comment (JellyBellyDev) [inline psalm.xml:10]: add also `spec` and `features` dir, make sanse?

File: composer.json [hunk 2]
```diff
@@ -31,15 +31,16 @@
...
+        "vimeo/psalm": "^3.12"
     },
     "config": {
         "bin-dir": "bin"
     },
     "bin": ["bin/poser"],
     "extra": {
         "branch-alias": {
-            "dev-master": "1.3.x-dev"
+            "dev-master": "1.4.x-dev"
```
Comment (JellyBellyDev) [inline composer.json:43]: revert please!

File: psalm.xml (hunk 1 above)
Comment (AlessandroMinoccheri) [reply psalm.xml:10]: Right! I have done it

**Comment by AlessandroMinoccheri** [comment]
I have rebase with `release\v2` now  @JellyBellyDev

**Comment by JellyBellyDev** [comment]
@AlessandroMinoccheri shit, I only remembered now to ask to update the changelog :S

**Comment by AlessandroMinoccheri** [comment]
new PR or can I push into this branch?

**Comment by JellyBellyDev** [comment]
New PR please! ;)

**Pull request merged**